sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -39,39 +39,43 @@ Use the returned token credential to authenticate the client.
 
 #### Create the client
 
-The data plane URI should be provided as the endpoint to create the client. 
+Azure Developer LoadTesting SDK has 2 clients to interact with the service, 'LoadTestRunClient' and 'LoadTestAdministrationClient'.
 
 ```python
-from azure.developer.loadtesting import LoadTestingClient
+from azure.developer.loadtesting import LoadTestAdministrationClient
 
 # for managing authentication and authorization
 # can be installed from pypi, follow: https://pypi.org/project/azure-identity/
 # using DefaultAzureCredentials, read more at: https://learn.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python
 from azure.identity import DefaultAzureCredential
 
-client = LoadTestingClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+client = LoadTestAdministrationClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
 ```
-## Key concepts
-
-The following components make up te Azure Load Testing service. The Azure Load Test client library for python allows you to interact with each of these components through the use of clients. There are two top-level clients which are the main entry points for the library
-
-- `LoadTestingClient` (`azure.developer.loadtesting.LoadTestingClient`)
+```python
+from azure.developer.loadtesting import LoadTestRunClient
 
-- Async `LoadTestingClient` (`azure.developer.loadtesting.aio.LoadTestingClient`)
+# for managing authentication and authorization
+# can be installed from pypi, follow: https://pypi.org/project/azure-identity/
+# using DefaultAzureCredentials, read more at: https://learn.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python
+from azure.identity import DefaultAzureCredential
 
-The two clients have similar methods in them except the methods in the async client are async as well.
+client = LoadTestRunClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
```

Comment:
Q: why did we move away from 2 subclients as was earlier?

Reply:
As per the API View comment, 

```Should the administration operations be a separate administration client?

If so, the two clients should flatten away these operation groups.


If we believe customers will use these together in a single client, I would recommend dropping 'load_test' from the operation group names, as it's already in the client name, and makes the names very long.``` by @annatisch 

The idea was that in C# we agreed to make 2 clients, so trying to be consistent.